app/src/main/java/com/sdp13epfl2021/projmag/model/TagsBaseManager.kt

```diff
@@ -78,4 +83,12 @@ import java.util.regex.Pattern
         return tags.toSet()
     }
 
+     fun tagsListToStringList(tags : Set<Tag>) : List<String>{
+         var listString = mutableListOf<String>()
+         for(tag in tags){
+             listString.add(tag.name)
+         }
+         return listString
```

Comment:
This could be simplified by
```suggestion
         return tags.toList().map(Tag::name)
```